content/docs/get-started/index.md

```diff
@@ -21,184 +23,142 @@ $ mkdir mlem-get-started
+
+<toggle>
+<tab title="Save new models">
+
+Just replace the line saving the model with a call to `mlem.api.save`:
+
+```py
+from mlem.api import save
+...
+
+# instead of joblib.dump(rf, "models/rf")
+save(rf, "models/rf", sample_data=df)
 ```
 
-Alongside the model binary `models/rf`, MLEM saved an additional metadata file
-`models/rf.mlem`. We refer to this as a "Codification" of the model. This
-`.mlem` "metafile" contains all the information we need in order to use the
-model later:
+The `sample_data` argument will prove useful later when serving the model. MLEM
+uses it to infer the data schema of the payload automatically!
 
-1. Model methods: Like `predict` and `predict_proba`
-2. Input data schema: Describes the dataframe (Iris dataset)
-3. Python Requirements: `sklearn` and `pandas` in this case, with the specific
-   versions used to train the model
+</tab>
 
-<admon type='tip'>
+<tab title="Import existing models">
 
-Note that we didn't have to specify any of this information ourselves. MLEM
-inspects the object (even if it's complex) and infers all of this automatically!
+You can use `mlem.api.import_object()` to "import" the object from Python code,
+which would create the the `rf.mlem` metadata file:
 
-</admon>
+```py
+import os
 
-<details>
+from mlem.api import import_object
 
-### Click to see the full contents of the `rf.mlem` metafile.
-
-```yaml
-artifacts:
-  data:
-    hash: 5a38e5d68b9b9e69e9e894bcc9b8a601
-    size: 163651
-    uri: rf
-model_type:
-  methods:
-    predict:
-      args:
-        - name: data
-          type_:
-            columns:
-              - sepal length (cm)
-              - sepal width (cm)
-              - petal length (cm)
-              - petal width (cm)
-            dtypes:
-              - float64
-              - float64
-              - float64
-              - float64
-            index_cols: []
-            type: dataframe
-      name: predict
-      returns:
-        dtype: int64
-        shape:
-          - null
-        type: ndarray
-    predict_proba:
-      args:
-        - name: data
-          type_:
-            columns:
-              - sepal length (cm)
-              - sepal width (cm)
-              - petal length (cm)
-              - petal width (cm)
-            dtypes:
-              - float64
-              - float64
-              - float64
-              - float64
-            index_cols: []
-            type: dataframe
-      name: predict_proba
-      returns:
-        dtype: float64
-        shape:
-          - null
-          - 3
-        type: ndarray
-  type: sklearn
-object_type: model
-requirements:
-  - module: sklearn
-    version: 1.1.2
-  - module: numpy
-    version: 1.22.4
-  - module: pandas
-    version: 1.5.0
+model_path = os.path.join(os.getcwd(), "models", "rf")
+target_path = os.path.join(os.getcwd(), "models", "rf.mlem")
+model = import_object(path=model_path, target=target_path, type_="pickle")
```

Comment:
```suggestion
model = import_object(path= "models/rf", target= "models/rf", type_="pickle")
```

Reply:
Oops, definitely needlessly verbose, fixed

Reply:
@aguschin shouldn't the target be  "models/rf.mlem" ?